package.json

```
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ignore-path .gitignore ."
```

Please update npm scripts to include a lint fix.

```suggestion
    "lint": "eslint --ignore-path .gitignore .",
    "lint:fix": "yarn lint --fix"
```